app/registration/views/patient-name-input.html

```diff
@@ -7,39 +7,89 @@ <h3 class="panel-title">{{'PATIENT_INFO_NAME' | translate}}</h3>
-                <input type="text" class="form-control keyboard-in" id="givenName" name="givenName" 
+                <input type="text" class="form-control keyboard-in" id="givenName" name="givenName"
                        ng-model="patient.givenName"
                        placeholder="{{'PATIENT_INFO_NAME_PLACEHOLDER' | translate}}" title="Enter patient's first name."
                        my-autocomplete source="getAutoCompleteList" response-map="getDataResults"
                        data-item-type="personName"
                        required>
             </div>
         </div>
+        <div class="form-group form-group-lg"
+             ng-class="{'has-error': aForm.patientSurname.$invalid && showMessages}">
+          <label for="patientSurname" class="col-sm-2 control-label">{{'PATIENT_INFO_SURNAME' | translate}}:<span class="asterick">*</span></label>
+          <div class="col-sm-10">
+            <div ng-messages for="aForm.patientSurname.$error"
+                 ng-messages-include="../poc-common/form-display/views/error-messages.html"
+                 ng-show="showMessages">
+            </div>
+            <input type="text" class="form-control keyboard-in" id="patientSurname" name="patientSurname"
+                   ng-model="patient.familyName"
+                   placeholder="{{'PATIENT_INFO_SURNAME_PLACEHOLDER' | translate}}" title="Enter patient's last name."
+                   required>
+          </div>
+        </div>
         <div class="form-group form-group-lg">
             <label for="patient_muddle_name" class="col-sm-2 control-label">{{'PATIENT_INFO_MIDDLE_NAME' | translate}}:</label>
             <div class="col-sm-10">
                 <input type="text" class="form-control keyboard-in" id="patient_muddle_name" ng-model="patient.middleName"
                        placeholder="{{'PATIENT_INFO_MIDDLE_NAME_PLACEHOLDER' | translate}}" title="Enter patient's middle name.">
             </div>
         </div>
-        <div class="form-group form-group-lg"
-             ng-class="{'has-error': aForm.patientSurname.$invalid && showMessages}">
-            <label for="patientSurname" class="col-sm-2 control-label">{{'PATIENT_INFO_SURNAME' | translate}}:<span class="asterick">*</span></label>
-            <div class="col-sm-10">
-                <div ng-messages for="aForm.patientSurname.$error" 
-                     ng-messages-include="../poc-common/form-display/views/error-messages.html"
-                     ng-show="showMessages">
-                </div>
-                <input type="text" class="form-control keyboard-in" id="patientSurname" name="patientSurname" 
-                       ng-model="patient.familyName"
-                       placeholder="{{'PATIENT_INFO_SURNAME_PLACEHOLDER' | translate}}" title="Enter patient's last name."
-                       required>
-            </div>
+
+      <div id="ng-similarpatients" ng-controller="SimilarPatients">
+        <script type="text/javascript">
```

Comment:
@valvijo agrreed! I will make changes to align to  angularjs as opposed to jquery